multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangahub/MangaHub.kt

```diff
@@ -330,6 +349,13 @@ abstract class MangaHub(
                         "MangaHub" -> "m01"
                         "MangaReader.site" -> "mr01"
                         "MangaPanda.onl" -> "mr02"
+                        "1Manga.co" -> "mn03"
+                        "MangaFox.fun" -> "mf01"
+                        "MangaHere.onl", "OneManga.info" -> "mh01"
+                        "Mangakakalot.fun" -> "mn01"
+                        "MangaNel" -> "mn05"
+                        "MangaOnline.fun" -> "m02"
+                        "MangaToday" -> "m03"
```

Comment:
probably better to make this constructor parameter/abstract property and have these sources override it, instead of putting them in the main class.

Reply:
Yeah